hasher-matcher-actioner/terraform/fetcher/variables.tf

```diff
@@ -47,10 +47,9 @@ variable "log_retention_in_days" {
   type        = number
 }
 
-variable "fetch_frequency_min" {
-  description = "How many minutes to wait between queries to ThreatExchange for updates"
-  type        = number
-  default     = 15
+variable "fetch_frequency" {
+  description = "How long to wait between calls to ThreatExcahnge. Must be an AWS Rate Expression. See here: https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html"
+  type        = string
```

Comment:
I ran into problems with this a few times, thanks for changing!

Reply:
Seems like a terrible AWS design decision that you have to say `1 minute` but `5 minute**s**` lol